src/modules/layout/containers/__tests__/__snapshots__/LocationLayout.spec.js.snap

```diff
@@ -11,8 +11,8 @@ exports[`LocationLayout should show CategoriesToolbar if current route is catego
   }
   header={
     <Connect(Translate(LocationHeader))
-      isEventsEnabled={false}
-      isExtrasEnabled={false}
+      isEventsEnabled={undefined}
```

Comment:
I think these should not be undefined

Reply:
kind of, but we don't need them for LocationLayout... I find it way more clear to read the tests if we don't specify all atributes of models even if we don't need them. Although we have undefined in our snaps then

Reply:
kind of, but we don't need them for LocationLayout... I find it way more clear to read the tests if we don't specify all atributes of models even if we don't need them. Although we have undefined in our snaps then